apps/aehttpserver/lib/aehttpserver/web/channels/notify.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Aehttpserver.Web.Notify do
+  alias Aeutil.Serialization
+
+  def broadcast_new_transaction_in_the_pool(tx) do
+    Aehttpserver.Web.Endpoint.broadcast!("room:notifications","new_tx:" <> Base.encode16(tx.data.from_acc), %{"body" => Serialization.tx(tx, :serialize)})
+    Aehttpserver.Web.Endpoint.broadcast!("room:notifications","new_tx:" <> Base.encode16(tx.data.to_acc), %{"body" => Serialization.tx(tx, :serialize)})
+    Aehttpserver.Web.Endpoint.broadcast!("room:notifications", "new_transaction_in_the_pool", %{"body" => Serialization.tx(tx, :serialize)})
+  end
+
+  def broadcast_new_block_added_to_chain_and_new_mined_tx(block) do
+    Enum.each(block.txs, fn(tx) -> 
+      Aehttpserver.Web.Endpoint.broadcast!("room:notifications", "new_mined_tx_everyone", %{"body" => Serialization.tx(tx, :serialize)})
```

Comment:
This looks like it's goint to produce tx_number of socket writes * listners_count whenever a block is mined. Isn't it too much?

Reply:
I think its ok for now, lets keep it like this